doc/api/deprecations.md

```diff
@@ -3056,6 +3056,22 @@ This method was deprecated because it is not compatible with
 
 Use [`buffer.subarray`][] which does the same thing instead.
 
+### DEP0159: `ERR_INVALID_CALLBACK`
+
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/41678
+    description: End-of-Life.
+-->
+
+Type: End-of-Life
+
+This error was removed due to adding more confusion to
+the errors used for value type validation.
+
+Use [`ERR_INVALID_ARG_TYPE`][] instead.
```

Comment:
I don't think we need to redirect user to a different error code, it's an internal thing after all.
```suggestion
```